### PR TITLE
update stats page to show both mathlib3 and mathlib4

### PR DESCRIPTION
**Pull request by robertylewis**
This has a perhaps more controversial change incorporated: I've removed the by-author graphs and tables. Personally I'm not sad to see them go. They've served their purpose. They're also harder to compile combined mathlib3/mathlib4 data. 

image
image
image
image

### CODE REVIEW

**Comment by PatrickMassot** [comment]
Do I read correctly that you don't explain at all the two colors? Could you add an explanation?

**Comment by robertylewis** [comment]
The tooltips explain it, but I've also just added a key at the top. The formatting may leave something to be desired.

image


image

File: templates/mathlib_stats.html [hunk 1]
```diff
@@ -52,19 +44,10 @@ <h2>Temporal distribution</h2>
...
-    </div>
-
-    <canvas id="lines_by_authors" class="mt-5"></canvas>
-
-      <canvas id="commits_by_authors"></canvas>
 
     <h2>Dependency graph</h2>
 
-    <p>A visualization showing how the various topics in mathlib interact and their relative sizes can be found <a href="https://eric-wieser.github.io/mathlib-import-graph">here</a>.</p>
+    <p>A visualization showing how the various topics in mathlib3 interact and their relative sizes can be found <a href="https://eric-wieser.github.io/mathlib-import-graph">here</a>.</p>
```
Comment (eric-wieser) [inline templates/mathlib_stats.html:50]: I guess I ought to rebuild this soon!

File: templates/mathlib_stats.html [hunk 2]
```diff
@@ -74,111 +57,222 @@ <h2>Dependency graph</h2>
...
-
+        });
+    }
+    
+    
+    window.onload = function() {
+        pad_mathlib4_data(by_year_month, by_year_month4)
+        pad_mathlib4_data(by_year, by_year4)
+        // pad_mathlib4_array('Number of lines', lines_by_date, lines_by_date4)
+        // pad_mathlib4_array('Number of files', files_by_date, files_by_date4)
```
Comment (eric-wieser) [inline templates/mathlib_stats.html:255]: What's your intent with these commented-out lines?

Comment (robertylewis) [reply templates/mathlib_stats.html]: Dead code. Deleted

**Pull request merged**